.cirrus.yml

```diff
@@ -1,10 +1,10 @@
 task:
-  name: rust 1.69 on freebsd 13
+  name: rust 1.74 on freebsd 13
```

Comment:
`OsString` is really only usable since 1.74, so a MSRV bump is necessary. This PR isn't urgent, so feel free to delay this PR to whenever such a MSRV bump would be warranted.

Reply:
Updating the MSRV is fine since it'll be part of a "major" release in any case since it's a breaking change.